src/config.cpp

```diff
@@ -97,10 +106,14 @@ void read_config_file(std::unordered_map<std::string, std::string>& umap,
         if (pos != std::string::npos) {
             std::string key = trim(line.substr(0, pos));
             std::string value = trim(line.substr(pos + 1));
+            // Skip if an env var with the same name exists.
+            if (getenv(get_clvk_env_name(key).c_str()) != nullptr) {
```

Comment:
This is not checking that the parsing of that env went well. If `parse_env` did not end up really setting the value, I don't think we should skip here.

I think it would be easier to just use `parse_env` twice. Once in each `init_config` functions.